### DISCUSSION TITLE
Eraser Tool for  Inpaint Outpaint

### DISCUSSION THREAD

**Discussion by paperclip52** [lllyasviel/Fooocus#3121] [General]
The latest update (2.4.3) is Great!.....BUT STILL NO ERASER TOOL FOR INPAINT OUTPAINT!!!!
I mean come on a SIMPLE eraser brush how hard can it be????  I've heard you have to get into Gradio
to implement this, so can't SOMEONE, ANYONE Accomplish this!?!?!?

**Comment by mashb1t** [comment]
What's the issue with your anger management, did you need an outlet and chose to use this repository to do so? (same for https://github.com/lllyasviel/Fooocus/discussions/3125)
Please value the time of all contributors (all done in spare time, not as part of a job) and be aware of that over 30 PRs have been done in the last 3 weeks.
If you want to be collaborative and contribute you can do so by continuing the work in the gradio update PR.

**Comment by PeterJared** [comment]
I used ChatGTP to translate it to the correct format :). Perhaps we should add a automatic translator in some cases (just joking) :)

"Hi there!

I wanted to start by saying that the latest update (2.4.3) is fantastic – great job! However, I did notice that there's still no eraser tool for inpaint/outpaint. A simple eraser brush would be incredibly useful and I'm sure many users would appreciate it.

I've heard that implementing this might require getting into Gradio, so I was wondering if there's any chance that someone could take this on? It would make a big difference!

Thanks so much for your hard work!"

**Reply by mashb1t** [reply]
Wow, thank you so much @paperclip52 for valuing the latest update of Fooocus, much appreciated.
Sadly we haven't yet had the time to look further into the feature, but it's in the backlog and has not been forgotten. It will be revisited at a later point in time, but if somebody takes over in the meantime we can proceed with the implementation even faster.
Thanks again for your heartwarming comment.

(thanks @PeterJared 👍)

**Comment by paperclip52** [comment]
I'm TRUELY SORRY for my bad conduct. Please except my deepest  APOLOGIES, to you and all the members. I was moving Fooocus from an internal ssd that was getting full from downloading lots of checkpoints an loras to an external hdd. For one reason or another the loras weren't loading (failure to initiate loading of lora folder?). I tried uninstalling and reinstalling a few times....I think I finally got everything in working order? And on top of all that a couple of my appliances quit working, you know the ole "produce to fail routine". Also lost my brother-in-law.  
Again, I'm truely sorry and do apologize.
Thanks Ever SoBrian
    On Sunday, June 9, 2024 at 05:25:40 AM EDT, Manuel Schmid ***@***.***> wrote:  
 
 


What's the issue with your anger management, did you need an outlet and chose to use this repository to do so? (same for #3125)
Please value the time of all contributors (all done in spare time, not as part of a job) and be aware of that over 30 PRs have been done in the last 3 weeks.
If you want to be collaborative and contribute you can do so by continuing the work in the gradio update PR.
